frr-tools/cp-tool/cp-tool.go

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cp "github.com/otiai10/copy"
+
+	"github.com/go-kit/log/level"
+	"go.universe.tf/metallb/internal/logging"
+)
+
+func main() {
+	logger, err := logging.Init("error")
+	if err != nil {
+		fmt.Printf("failed to initialize logging: %s\n ", err)
+		os.Exit(1)
+	}
+
+	if len(os.Args) != 3 {
+		level.Error(logger).Log("Please provide two command line arguments.")
+		return
+	}
+
+	sourceFile, destinationFile := os.Args[1], os.Args[2]
+	sourceFile = filepath.Clean(sourceFile)
+	destinationFile = filepath.Clean(destinationFile)
+
+	err = cp.Copy(sourceFile, destinationFile)
```

Comment:
did you try something that doesn't require importing another dependency? I wonder if we can just do this with some `io.Copy` (or anything else from the standard library) that wouldn't be too messy since our usecase is pretty simple

Reply:
Originally there was file permission issue for the local copy tool when using io.Copy, so I switched to a community package. With the new submission, I've added the preserve permission functionality, so it should be clean now.